src/command.ts

```diff
@@ -0,0 +1,43 @@
+import { Command } from './types'
+
+const commandParse: RegExp = /^(?:(\w+)\.)?(?:(\w+):)?(\w+)$/
```

Comment:
This regexp can parse string, like `foo.bar:piyo`,  to array, `['foo.bar.piyo', 'foo', 'bar', 'piyo']`.